extension/js/common/composer.ts

```diff
@@ -508,7 +508,9 @@ export class Composer {
       if (result.success) {
         this.S.cached('prompt').css({ display: 'none' });
         Xss.sanitizeRender(this.S.cached('input_text'), await Xss.htmlSanitizeKeepBasicTags(result.content.toUtfStr().replace(/\n/g, '<br>')));
-        if (headers && headers.to && headers.to.length) {
+        if (this.urlParams.isReplyBox) {
+          await this.renderReplyMsgComposeTable();
+        } else if (headers && headers.to && headers.to.length) {
           this.S.cached('input_to').focus();
```

Comment:
Thanks for the PR. Looking at the code I would imagine that loading recipients from the draft would no longer work, and instead it will load recipient based on the previous message.

I haven't tested it, but you can:
1) click reply
2) start typing draft
3) add a new recipient
4) wait for draft to save
5) refresh page, wait for draft to load
6) I think the new recipient will be missing

Please test this scenario and please fix it if it needs fixing. If it works well, let me know so I can merge it as is.

Thanks!

Reply:
Yes, Everything works well. Every Recipient is loaded.

Reply:
Perfect, thanks!